README.md

```diff
@@ -4619,6 +4632,301 @@ The command above assumes you mounted the data workspace in `/mount/data`, and t
+&& export HYDRA_FULL_ERROR=1 \
+&& python rlhf/rlhf_nemo/serve_initial_policy.py \
+    --config-path=/opt/nemo-rlhf/rlhf/rlhf_nemo/conf \
+    --config-name=inference_initial_policy \
+    gpt_model_file=/path/to/model.nemo \
+    port=5556
+```
+
+This command will launch the Initial Policy inference server on the local computer, using port 5556. All the configuration parameters can be modified in the inference_initial_policy.yaml file, or by overriding them through the CLI command. Ensure server=True is set in the configuration of this job to correctly launch the inference server.
+
+Note: data parallelism is not available for the inference servers, so only a single copy of the model will be available
+
+##### 5.14.2.3 Launching the PPO Critic Training and Inference Server
+<a id="markdown-launching-the-ppo-critic-training-and-inference-server" name="launching-the-ppo-critic-training-and-inference-server"></a>
+
+The PPO Critic has to perform both training and inference. We designed the Critic to have both capabilities. To launch the PPO Critic server in a Linux system, this command can be run inside the container:
+
+```bash
+cd /opt/nemo-rlhf \
+&& export PYTHONPATH="/opt/nemo-rlhf:${PYTHONPATH}" \
+&& export HYDRA_FULL_ERROR=1 \
+python rlhf/rlhf_nemo/serve_ppo_critic.py \
+    --config-path=/opt/nemo-rlhf/rlhf/rlhf_nemo/conf \
+    --config-name=gpt_ppo_critic \
+    model.pretrained_checkpoint.restore_from_path=/path/to/trained_rm.nemo \
+    port=5557
+```
+
+This command will launch the PPO Critic server on the local computer, using port 5557. All the configuration parameters can be modified in the gpt_ppo_critic.yaml file, or by overriding them through the CLI command. Ensure inference.server=True is set in the configuration of this job to correctly launch the server.
+
+Note: data parallelism is not available for the servers, so only a single copy of the model will be available.
+
+##### 5.14.2.4 Launching the PPO Actor Training
+<a id="markdown-launching-the-ppo-actor-training" name="launching-the-ppo-actor-training"></a>
+The PPO Actor training job contains the master HTTP controller that makes the HTTP calls to all three servers when needed. To launch the PPO Actor server in a Linux system, this command can be run inside the container:
+
+```bash
+cd /opt/nemo-rlhf \
+&& export PYTHONPATH="/opt/nemo-rlhf:${PYTHONPATH}" \
+&& export HYDRA_FULL_ERROR=1 \
+&& python rlhf/rlhf_nemo/train_gpt_ppo_actor.py \
+    --config-path=/opt/nemo-rlhf/rlhf/rlhf_nemo/conf \
+    --config-name=gpt_ppo_actor \
+    "model.data.data_prefix={train: [/path/to/train_data], validation: [/path/to/val_data], test: [/path/to/test_data]}" \
+    model.pretrained_checkpoint.restore_from_path=/path/to/model.nemo
+```
+This command will launch the PPO Actor job on the local computer. All the configuration parameters can be modified in the gpt_ppo_actor.yaml file, or by overriding them through the CLI command.
```

Comment:
Consider highlighting `gpt_ppo_actor.yaml`
Is there no server=True parameter here?